conda/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set data= load_setup_py_data() %}
+
+package:
+    name: ts-nightlydigest
```

I think the package defined here must be the same as the one in the `Jenkinsfile.conda` file. So:
```suggestion
    name: ts-logging-and-reporting
```

---

Please also take a look to the CI tests :pray:.